.github/workflows/old-test.yml

```diff
@@ -30,17 +30,9 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: Cache Node dependencies
-        uses: actions/cache@v3
-        with:
-          path: ~/.npm
-          key: ${{ runner.os }}-node-${{ hashFiles('**/package.json') }}
-          restore-keys: |
-            ${{ runner.os }}-node-
-
       - name: Install dependencies
-        run: npm install
+        run: npm install --ignore-scripts --production
```

Comment:
Why do we now need to have `--ignore-scripts` there?
Did you find what caused the issue on `master` branch?

Also, should we set `ignore-scripts=true` in `.npmrc` at the root (as you're now setting this option everywhere when running `npm install`)?

Reply:
I removed the `--ignore-scripts` now.

I added it as an hypothesis that it could be the cause. Then kept it due to best practice. But you're right, its out of scope.